src/index.js

```diff
@@ -232,6 +232,8 @@ class TerserPlugin {
             if (this.options.cache) {
               const defaultCacheKeys = {
                 terser: terserPackageJson.version,
+                'terser_no_bundle_env': process.env.TERSER_NO_BUNDLE,
```

Comment:
Hm, why we need here `process.env.TERSER_NO_BUNDLE`? Can you example?

Reply:
@fabiosantoscode i find this variable only here https://github.com/terser-js/terser/blob/9b98031ccc380f6359915176ce7a622113323af6/bin/uglifyjs, but we don't use `bin`

Reply:
I think I might need to add TERSER_NO_BUNDLE to terser-webpack-plugin then, instead of requiring 'terser' you would require 'terser/tools/node' to avoid using the bundle.

TERSER_NO_BUNDLE is for when you're debugging a terser error and want to edit the terser code directly by adding eg console.logs to it

Reply:
@fabiosantoscode let's do it in other PR